thorbanks/utils.py

```diff
@@ -162,16 +173,17 @@ def verify_signature(request, bank_name, signature, auth=False, response=False):
 
     digest = request_digest(request, bank_name, auth=auth, response=response)
 
-    try:
-        public_key.verify(
-            b64decode(signature), digest, padding.PKCS1v15(), hashes.SHA1()
-        )
-
-        return True
-
-    except InvalidSignature:
-        return False
+    hash_algorithms = [hashes.SHA1(), hashes.SHA256()]
```

Comment:
Not the best solution with the hash algorithms defined in two places. Could move these to utils.py